docs/ARCHITECTURE.rst

```diff
@@ -755,16 +759,16 @@ command in the Zenko CloudServer directory:
 
 This will open two ports:
 
--  one is based on socket.io and is used for metadata transfers (9990 by
+- one is based on socket.io and is used for metadata transfers (9990 by
   default)
 
--  the other is a REST interface used for data transfers (9991 by
+- the other is a REST interface used for data transfers (9991 by
   default)
 
 Then, one or more instances of Zenko CloudServer without the dmd can be started
 elsewhere with:
 
-::
+.. code:: sh
```

Comment:
not sure if this matters, but in the 'CLIENTS.rst' file, this line reads 
`.. code-block:: shell`
and the following command starts with a `$ `

Reply:
I guess `code-block` is for multiple lines. `$` point still stands tho :)